.coveragerc

```diff
@@ -1,2 +1,8 @@
 [run]
 branch = True
+
+[paths]
+source =
+    src/
+    .tox/py*/lib/python*/site-packages/
```

Comment:
This could probably use some explanation (I don't know if you can put comments in this file or not), but it would help to try to explain this and why there are two entries. Is it a `venv` / `virtualenv` thing? Windows/*Nix?